app/jobs/run_submissions_job.rb

```diff
@@ -0,0 +1,28 @@
+class RunSubmissionsJob < ApplicationJob
+  queue_as :submission
```

Comment:
To differentiate this from the Submission model, could we call the queue something like `:pending_submission`?